getting-started/templates/systemlink-values.yaml

```diff
@@ -987,6 +1014,133 @@ fileingestion:
+    username: "filescdc"
+
+  ## High Availability configuration
+  ##
+  highAvailability:
+    ## Whether to enable high availability. This is required for production environments.
+    ## Needs an object storage medium, like S3, Azure Storage or a ReadWriteMany volume.
+    enabled: true
+    ## Storage configuration
+    ##
+    storage:
+      ## The type of the object storage medium. Supported values are "s3", "azure" and "readWriteManyVolume"
+      type: "s3"
+      ## Configure S3/MinIO access.
+      s3:
+        ## Secret name for S3 credentials
+        secretName: "flink-aws-keys"
+        ## S3 access type. Supported values are "ACCESS_KEY" and "AWS_WEB_IDENTITY_TOKEN".
+        authType: "ACCESS_KEY"
+        ## The name of the S3 bucket that the service should connect to.
+        bucket: "systemlink-flink"
+        ## Set this value to connect to an external S3 instance.
+        host: "s3.amazonaws.com"
+        ## Set this value to connect to an S3 instance which is internal to the cluster.
+        service: ""
+        ## S3 Region.
+        region: "us-east-1"
+        ## The name of the key in the above secret whose value contains the S3 AccessKeyId.
+        accessKeyIdKey: "aws-access-key-id"
+        ## The name of the key in the above secret whose value contains the S3 SecretAccessKey.
+        secretAccessKeyKey: "aws-secret-access-key"
+      ## Configure Azure Storage access.
+      azure:
+        ## The name of the Azure Storage account. Required if Azure Storage is enabled.
+        accountName: ""
+        ## The name of the secret containing the Azure Storage credentials.
+        secretName: "flink-azure-credentials"
+        ## The name of the Azure Storage container that the service should connect to.
+        containerName: "systemlink-flink"
+        ## The name of the key in the above secret whose value contains the shared access key.
+        accessKeyName: "azure-secret-access-key"
+        ## Whether to use TLS for connections to Azure Storage.
+        isSecure: true
+        ## The Azure Blob Storage host and port to connect to, not including the account name.
+        blobApiHost: "blob.core.windows.net"
+      ## Configure ReadWriteManyVolume access.
+      readWriteManyVolume:
+        persistentVolume:
+          ## The name of the kubernetes storage class that supports ReadWriteMany volumes.
+          storageClassName: "efs-sc"
+
+  ## Job configuration.
+  ##
+  job:
+    ## Connectors configuration.
+    ##
+    connectors:
+      ## Sink configuration.
+      sink:
+        ## Elasticsearch sink configuration.
+        elasticsearch:
+          ## Index creation configuration.
+          index:
+            ## The number of primary shards that will be allocated for the files elastic index.
+            ## This can be configured only on the first deployment, subsequent deployments will ignore this value.
+            primaryShardsCount: 2
+
+  ## Job Manager configuration.
```

Comment:
I think we've set good defaults that probably won't be changed so we can omit this section